app/assets/stylesheets/styles.scss

```diff
@@ -67,6 +67,10 @@ h3 {
   &.sectionHead {
```

Comment:
the `h3.sectionHead`'s under How It Works seem like they have a ridiculous amount of padding.